### PR TITLE
Implement a widget for GitLab CI build status

### PR DESCRIPTION
**Pull request by splattael**

### CODE REVIEW

**Comment by splattael** [comment]
@jnbt I need some :eyes: on this please :yellow_heart:

**Comment by jnbt** [comment]
💖

**Comment by jnbt** [comment]
I'm more than happy to merge it, once you think it is ready.

File: lib/neoboard/pusher.ex [hunk 1]
```diff
@@ -20,12 +20,12 @@ defmodule Neoboard.Pusher do
     namespace = Atom.to_string(module)
     |> String.split(".")
     |> List.last
-    |> String.downcase
+    |> Macro.underscore
```
Comment (splattael) [inline lib/neoboard/pusher.ex:23]: @jnbt WDYT? Do you agree?

Comment (jnbt) [reply lib/neoboard/pusher.ex:23]: Agree...

**Comment by splattael** [comment]
@jnbt I've added some tests and squashed into a single commit.

:shipit: ?

**Comment by splattael** [comment]
Travis fails due to `mix deps.get` errors. Strange. I'll have a look into it.

**Comment by splattael** [comment]
Travis failure solved.

**Comment by splattael** [comment]
Ready to merge! Need a :+1: :D

**Pull request merged**